### PR TITLE
`Performance/CollectionLiteralInLoop` should be `Enabled: 'pending'`

### PR DESCRIPTION
**Pull request by ghiculescu**
As noted [here](https://github.com/rubocop-hq/rubocop-performance/pull/140#discussion_r487437198).

### CODE REVIEW

**Comment by koic** [comment]
Sure! Can you update the related doc using `rake generate_cops_documentation` and add a changelog entry?

**Comment by ghiculescu** [comment]
@koic done!

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## master (unreleased)
 
+* [#181](https://github.com/rubocop-hq/rubocop-performance/pull/181): Change default configuration for `Performance/CollectionLiteralInLoop` to `Enabled: 'pending'`. ([@ghiculescu][])
```
Comment (koic) [inline CHANGELOG.md:5]: This entry looks good! Only one more point, can you move the entry to under the `### Changes` section?

Comment (ghiculescu) [reply CHANGELOG.md]: ooops, done.

Comment (ghiculescu) [reply CHANGELOG.md]: Sorry about that.

Comment (koic) [reply CHANGELOG.md]: Thank you for updating!

**Pull request merged**